### PR TITLE
Update README with TRELLIS_INSTALL_PREFIX example

### PR DESCRIPTION
**Pull request by gojimmypi**
see https://github.com/YosysHQ/nextpnr/pull/404 and https://github.com/SymbiFlow/prjtrellis/pull/123

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,7 +97,8 @@ such as pack, place, route, and write output files.)
 ### nextpnr-ecp5
 
 For ECP5 support, you must download [Project Trellis](https://github.com/SymbiFlow/prjtrellis),
-then follow its instructions to download the latest database and build _libtrellis_.
+then follow its instructions to download the latest database and build _libtrellis_. 
+(for example: `-DTRELLIS_INSTALL_PREFIX=/usr` will install to `/usr/share/trellis`)
```
Comment (daveshah1) [inline README.md]: `TRELLIS_INSTALL_PREFIX` doesn't install to anywhere, it is the prefix where trellis is installed to. And trellis isn't just installed to `/usr/share` as it puts the compiled library in `/usr/lib` too

Comment (gojimmypi) [reply README.md]: ah. so the text in my example should be "will **look** here" ?

Comment (daveshah1) [reply README.md]: Please add 'and `/usr/lib/trellis`' at the end, then it is good

Comment (gojimmypi) [reply README.md]: done

**Pull request merged**